jenkins/edx_platform_test_notifier.py

```diff
@@ -83,8 +83,16 @@ def notify_tests_completed(self, pr):
         else:
             logger.info("Successfully commented on PR.")
 
+    def get_head_commit(self, pr):
```

Comment:
Maybe add a docstring with some of the context from the PR description about how there can be no commits?

Reply:
done